paragon/_variables.scss

```diff
@@ -0,0 +1,233 @@
+$limestone-500: #D7D3D1; // 300
+$limestone-300: #EBE9E9; // 250
+$limestone-200: #F5F4F4; // 150
+$limestone-100: #FAF9F7; // 100
+$isotope-blue: #03C7E8;
+$oxide-yellow: #F0CC00;
+$text-gray: #454545;
+$text-elm: #2D494E;
+
+// Color system
+$white:    #fff !default;
+$gray-100: $limestone-200 !default;
+$gray-200: $limestone !default;
+$gray-300: $limestone-500 !default;
+$gray-500: #686B73 !default;
+$gray-700: #2D323E !default;
+$gray-900: #171C29 !default;
+$black:    #000 !default;
+
+$grays: () !default;
+$grays: map-merge(
+  (
+    "100": $gray-100,
+    "200": $gray-200,
+    "300": $gray-300,
+    "500": $gray-500,
+    "700": $gray-700,
+    "900": $gray-900
+  ),
+  $grays
+);
+
+$blue: #23419f !default;
+$red: #C32D3A !default;
+$yellow: $oxide-yellow !default;
+$green: #178253 !default;
+$teal: #006DAA !default;
+
+$colors: () !default;
+$colors: map-merge(
+  (
+    "blue": $blue,
+    "green": $green,
+    "teal": $teal,
+    "red": $red,
+    "yellow": $yellow,
+    "white": $white,
+    "gray": $gray-500,
+    "dark-gray": $gray-700,
+  ),
+  $colors
+);
+
+$primary: $elm !default;
+$brand: $garnet !default;
+$success: $green !default;
+$info: $teal !default;
+$danger: $red !default;
+$warning: $yellow !default;
+$light: #eee !default;
+$dark: $gray-700 !default;
+
+$theme-colors: () !default;
+$theme-colors: map-merge(
+  (
+    "primary":         $primary,
+    "brand":           $brand,
+    "success":         $success,
+    "info":            $info,
+    "warning":         $warning,
+    "danger":          $danger,
+    "light":           $light,
+    "dark":            $dark,
+  ),
+  $theme-colors
+);
+
+
+$theme-color-levels: () !default;
+
+$theme-color-levels: map-merge(
+  (
+    "gray-100": $gray-100,
+    "gray-200": $gray-200,
+    "gray-300": $gray-300,
+    "gray-500": $gray-500,
+    "gray-700": $gray-700,
+    "gray-900": $gray-900,
+
+    // "primary-100": #f4f7ff,
+    // "primary-200": #D8EDF8,
+    // "primary-300": #6c83cd,
+    // "primary-500": $primary,
+    // "primary-700": #1a337b,
+    // "primary-900": #03195c,
+
+    "success-100": #F2FAF7,
+    "success-200": #BBE6D7,
+    "success-300": #30A171,
+    "success-500": $success,
+    "success-700": #175B3C,
+    "success-900": #0F4D0F,
+
+    "info-100": #EFF8FA,
+    "info-200": #9CD2E6,
+    "info-300": #1C8DBE,
+    "info-500": $info,
+    "info-700": #004972,
+    "info-900": #002F4A,
+
+    "danger-100": #FCF1F4,
+    "danger-200": #F3A9BE,
+    "danger-300": #E4406E,
+    "danger-500": $danger,
+    "danger-700": #942947,
+    "danger-900": #691B30,
+
+    // "warning-100": #fffaed,
+    // "warning-200": #FFD875,
+    // "warning-300": #FFBF18,
+    // "warning-500": $warning,
+    // "warning-700": #996B00,
+    // "warning-900": #7A5500,
+  ),
+  $theme-color-levels
+);
+
+
+// Typography
+
+$font-family-sans-serif:      "Inter", "Helvetica Neue", Arial, sans-serif !default;
+$font-family-serif:           "Georgia", serif !default;
```

Comment:
I thought we were getting rid of Georgia?

Reply:
We don't use the serif font anywhere that I'm aware of but Bootstrap does have this serif variable. So it seems like we should specify even if it's unused. We could remove Georgia and leave it at 'serif';